### DISCUSSION TITLE
Get all options of a select fieldtype

### DISCUSSION THREAD

**Discussion by pathly-dev** [statamic/cms#5221] [Q&A] [answered]
Hey there,

I have kind of a blog where I have a select fieldtype for each blog post. And now I also want to show all of the select options on the template on the blog page, so the user can filter for specific things. Unfortunately the documentation says pretty nothing what you can do with selects. So I dont get how to get all the options to display on my website. Hopefully someone can help.

Thanks!

**Comment by jackmcdade** [comment, marked as answer]
Sounds like a good use case for Taxonomies!

**Reply by pathly-dev** [reply]
Ah thanks, haven't used taxonomies so far. Seems exactly like what I was looking for. Thanks!
